affinity/cpp-23/d1795r2.md

```diff
@@ -110,29 +113,59 @@ Another example of this is that while Hwloc is highly used in many domains, it n
 
 # 5. Proposed direction
 
+## Overview
+
 This paper aims to build on the unified executors proposal, detailed in P0443 [[14]][p0443r11], so this proposal and any others that stem from it will target P0443 as a baseline, and aim to integrate with its direction as closely as possible.
 
-Below we outline a proposed direction:
+This paper proposes an interface which provides abstract representation of a system's resources and their connections and various properties, capabilities and limitations. This abstract representation will allow a diverse range of architectures both available now and to come in the future to be represented in C++ but without tying down the C++ abstract machine specific hardware definitions such as CPU, GPU, etc.
```

Comment:
@AerialMantis sorry I'm a bit late to this -- a minor suggested rewording:

This paper proposes an interface which provides an abstract representation of a system's resources and their connections, and various properties, capabilities and limitations. This abstract representation will allow a diverse range of architectures both available now and to come in the future to be represented in C++, but without tying the C++ abstract machine to specific hardware definitions like "CPU" or "GPU."